src/nspawn/nspawn.c

```diff
@@ -1018,6 +1024,15 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_settings_mask |= SETTING_READ_ONLY;
                         break;
 
+                case ARG_AMBIENT_CAPABILITY: {
+                        uint64_t m;
+                        r = parse_capability_spec(optarg, &m);
+                        if (r <= 0)
+                                return r;
+                        arg_caps_ambient = m;
```

Comment:
hmm, i think this should OR things in. In almost all cases our command line switches extend each other if that makes any conceptual sense, and hence we should do the same here.